FAQ.md

```diff
@@ -51,10 +51,26 @@ Please see the [TOC Principles](https://github.com/cncf/toc/blob/main/PRINCIPLES
 
 The CNCF provides a variety of services that are accessible by maintainers via the ServiceDesk: https://github.com/cncf/servicedesk
 
-## What is the definition of an end user?
-End user companies use [cloud native](https://github.com/cncf/toc/blob/main/DEFINITION.md) technologies internally, but do not sell any cloud native services externally. Examples of end user companies are Adidas, Apple, Capital One, Spotify, and The New York Times. Examples of cloud native vendors (and so not end users) include cloud providers (e.g., Alibaba Cloud, AWS, Google Cloud, Microsoft Azure), consulting firms (e.g., Accenture, Booz Allen), infrastructure software vendors (e.g., SUSE, Red Hat), and telecom operators (e.g., AT&T, China Mobile).
-
-If you’re not sure if your company is an end user, please compare the members of the [CNCF end user community](https://landscape.cncf.io/enduser=yes&format=card-mode) to the [vendor members](https://landscape.cncf.io/category=cncf-members&enduser=no&format=card-mode&grouping=category) of CNCF and find your company or one similar to yours in one group or the other. Or, you can email info@cncf.io and we’ll help you.
+## What is the definition of an adopter?
+Adopters of a CNCF project are any organization that successfully leverages that project in the manner it was intended or repackages it as a core component of a service offering.
+
+The TOC’s intent of identifying adopters is to better understand the operational or production-level use of a given project (which can also be a specification) by its operators or users. We do this to ascertain the level of maturity the project has reached, its interactions with adopters, and its likelihood to continue growing within and supporting the ecosystem.
+
+[Cloud native](https://github.com/cncf/toc/blob/main/DEFINITION.md) project adopters may be any one of the following or a combination thereof : 
+* A CNCF End-User member - Companies and organizations who are members of the CNCF (https://www.cncf.io/enduser/ & https://landscape.cncf.io/card-mode?enduser=yes)
+* Another project - an open source project that leverages a CNCF project as part of their solution, integrates with for compatibility and interoperability, or is used in the supply chain of another project
+* A Service Provider - a service provider is a organization that repackages an open source project as a core component of a service offering, sells cloud native services externally
```

Comment:
```suggestion
* A Service Provider - a Service Provider is a organization that repackages an Open Source project as a core component of a service offering, sells Cloud Native services externally
```

Reply:
I am unsure on capitalizing Cloud Native.

Reply:
i leverage https://github.com/cncf/toc/blob/main/DEFINITION.md for capitalization of cloud native.

Reply:
@RichiH  do we traditionally capitalize "open source"?

Reply:
Not usually.

Reply:
I thought we did; thanks for checking, closing.